CDCgov/NEDSS-Modernization
# CNFT1-1532 added auto slash for date

## Description

Added auto slash for date using JS since the trussworks didn't provide any masked input for dates

## Tickets

* https://cdc-nbs.atlassian.net/browse/CNFT1-1533

## Steps to verify

1. Manually type numbers in date picker it should automatically add slashes to validate date.
